test/_helpers.test.js

```diff
@@ -22,15 +20,8 @@ describe('Helpers', () => {
     expect(Object.keys(objects).length).toBe(2);
   });
 
-  test('torchToTF(): can convert underscore function', () => {
-    expect(torchToTF('__add__', {}, logger)).toBe('add');
-  });
-
   test('torchToTF(): can convert torch function', () => {
```

Comment:
That can be dropped together with torchToTf helper function.
We don't need to test what's already tested in threepio itself :)